README.md

```diff
@@ -5,40 +5,37 @@
 - **Field Name Prefix:** storage
 - **Scope:** Item, Asset
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
-- **Owner**: @davidraleigh
+- **Owner**: @davidraleigh @matthewhanson
 
 This document explains the Storage Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
 This extension adds fields to STAC Item and Asset objects, allowing for details related to cloud storage access and costs to be associated
 with a STAC Item.  This extension does not cover NFS solutions provided by PaaS cloud companies.
 
 - Examples:
-  - [Item example 1](examples/example-naip.json): Shows the basic usage of the extension in a STAC Item.
-  - [Item example 2](examples/example-nsl.json): Another example of basic usage.
+  - [Item example 1](examples/item-naip.json): Shows the basic usage of the extension in a STAC Item.
+  - [Item example 2](examples/item-nsl.json): Another example of basic usage.
 - [JSON Schema](json-schema/schema.json)
 - [Changelog](./CHANGELOG.md)
 
 ## Item Fields
+
 | Field Name  | Type   | Description |
 | ----------- | ------ | ----------- |
 | storage:min_tier_duration   | integer  | number of days for the shortest time tier restriction on access of an asset. |
 | storage:max_tier_duration   | integer  | number of days for the longest time tier restrictions on access of an asset. |
 | storage:archived            | bool    | descriptor for whether the data is "properly" archived according to implementers discretion |
-
-## Asset Fields
```

Comment:
Does this mean everything is under Item now?

Reply:
@davidraleigh Assets can now specify any STAC property that can be at the Item level, see https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#additional-fields-for-assets
So this allows any asset to use these properties as well. By having them at the Item level you can specify a single default value, but can override in any of the assets.

Reply:
I just saw this: ["While these are all valid properties on an Item, they will typically be defined per-asset. If a field applies equally"](https://github.com/stac-extensions/storage/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R36)  

I didn't know that was possible. cool